gui/integration_test/e2e_tests/legal_information.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:nordvpn/i18n/strings.g.dart';
+
+import '../../test/utils/test_helpers.dart';
```

Comment:
Can we extract test_helpers into an internal package so similar imports would be less fragile in the future?
(possibly, a question to @devzbysiu here)

Reply:
So instead of `import '../../test/utils/test_helpers.dart';` we'd able to import this way:
`import 'package:nordvon/test/utils/helpers.dart`;`

My gripe was about using relative paths that gets easily broken.

Reply:
Sorry for the delay. Yeah, it could be done, but then I think those test utils would have to be under `lib/` and not under `test` (`integration_test/` also won't work), so not sure if it makes sense to put test related code into non-test directory. 
The code won't be included in the final app, so that's not an issue - the only thing here is this test code in non-test dir. Then considering that this relative path would be an issue only when we would move the code (which we are not planning), I'm not sure if it makes sense to spend time on this. I wouldn't do that, but I'm also not objecting, so if you prefer it, we can have a task for it and address it